incubator/hnc/internal/kubectl/tree.go

```diff
@@ -64,6 +64,9 @@ var treeCmd = &cobra.Command{
 			fmt.Println(txt)
 			printSubtree("", hier, cycle)
 		}
+
+		fmt.Printf("[s] indicates subnamespaces.\n")
+
```

Comment:
We only want to print this if there _are_ subnamespaces. See the logic just below (`if len(footnotes) > 0`) as an example - but note that after your change, not all footnotes will be conditions anymore, so you might want to change some of the existing logic too.

Reply:
done